cmd/kogito/command/deploy/delete_service.go

```diff
@@ -69,32 +66,22 @@ func (i *deleteServiceCommand) Command() *cobra.Command {
 func (i *deleteServiceCommand) InitHook() {
 	i.flags = deleteServiceFlags{}
 	i.Parent.AddCommand(i.command)
-	i.command.Flags().StringVarP(&i.flags.project, "project", "p", "", "The project name")
+	i.command.Flags().StringVarP(&i.flags.project, "project", "p", "", "The project name from where the service needs to be deleted")
 }
 
-func (i *deleteServiceCommand) Exec(cmd *cobra.Command, args []string) error {
+func (i *deleteServiceCommand) Exec(cmd *cobra.Command, args []string) (err error) {
 	log := context.GetDefaultLogger()
 	i.flags.name = args[0]
-	var err error
 	if i.flags.project, err = shared.EnsureProject(i.Client, i.flags.project); err != nil {
 		return err
 	}
-
-	if err := shared.CheckKogitoAppExists(i.Client, i.flags.name, i.flags.project); err != nil {
+	if err = service.DeleteRuntimeService(i.Client, i.flags.name, i.flags.project); err != nil {
 		return err
 	}
-	log.Debugf("About to delete service %s in namespace %s", i.flags.name, i.flags.project)
-
-	if err := kubernetes.ResourceC(i.Client).Delete(&v1alpha1.KogitoApp{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      i.flags.name,
-			Namespace: i.flags.project,
-		},
-	}); err != nil {
-		return err
+	if i.Client.IsOpenshift() {
+		if err = service.DeleteBuildService(i.Client, i.flags.name, i.flags.project); err != nil {
+			log.Errorf("%s", err)
+		}
 	}
-
-	log.Infof("Successfully deleted Kogito Service %s in the Project %s", i.flags.name, i.flags.project)
-
```

Comment:
Since you removed this log info, is there another info about this command in another part of the command?

Reply:
From `delete-service` we are calling two different services `delete build` and `delete runtime` both of them have their success and failure msg